src/features/collaborator-avatars/CollaboratorAvatarItem.js

```diff
@@ -45,13 +47,13 @@ const CollaboratorAvatarItem = (props: Props) => {
             <Avatar name={name || '-'} {...rest} isExternal={isExternalCollab} shouldShowExternal={allowBadging} />
         );
 
+    const expirationTooltipMessage = intl.formatMessage(messages.expirationTooltipText, {
+        date: expiration?.executeAt,
```

Comment:
If this is undefined, do we want to show a different Tooltip Message?

Reply:
It will not even show up if it's undefined
`const expirationBadge =
        allowBadging && expiration && expiration.executeAt ? (`